### DISCUSSION TITLE
How can export the attributes to DesignExplorer?

### DISCUSSION THREAD

**Discussion by anjitecture** [Tunny-gh/Tunny_in_Open_Sea#347] [General]
Hi, as title. I want to use designexplorer but with objectives that I want as outputs that do not need to be optimised, they are just the statistics of the iteration. Attrbiutes are not showing in designexplorer, only variables and objectives.

**Comment by hrntsm** [comment]
Hi @anjitecture,

Attribute output is not supported in current version.
I add this feature request to #350

**Reply by hrntsm** [reply]
Fix in #350
